core/db/migrate/20250225051308_add_vat_id_email_and_reverse_charge_status_to_addresses.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddVatIdEmailAndReverseChargeStatusToAddresses < ActiveRecord::Migration[7.2]
```

Comment:
we still support rails 7.0. so please change the schema version to 7.0

Reply:
Done